types/setting.go

```diff
@@ -1112,6 +1115,16 @@ var (
 		ReadOnly:    false,
 		Default:     "5",
 	}
+
+	SettingDefinitionLogLevel = SettingDefinition{
+		DisplayName: "Log Level",
+		Description: "The log level 0 (Panic) ~ 6 (Trace) used in longhorn manager. By default 3 (Warning).",
```

Comment:
Should the default level be `warning` rather than `info`?

Reply:
`info` is lower than `warning`, so `warning` will have less logs than info level.

Reply:
`info` logs contain some important info like object state change. Suppressing it by default would make the support bundle debugging more difficult.

Reply:
@shuo-wu do you mean it will be more tedious? however, right now it's `debug` level which is even lower than `info`.

Reply:
I agree with @shuo-wu 's comment
Most of the people use default setting
So if they generate support bundle when encountering some issue, there will be no info message in the log and make us more difficult to debug

Reply:
Change to default info level

Reply:
I thought it was info. Yes we should use info.